packages/filters/filter-fxaa/src/fxaa.frag

```diff
@@ -113,11 +114,11 @@ vec4 fxaa(sampler2D tex, vec2 fragCoord, vec2 resolution,
 
 void main() {
 
-      vec2 fragCoord = vTextureCoord * filterArea.xy;
+      vec2 fragCoord = vTextureCoord * destinationFrame.ba;
```

Comment:
Please dont use rgba fields for non-colors.

Reply:
fair point :)

Reply:
done!